### PR TITLE
Extend authz list with LIGO DNs for all LIGO private namespaces in cache Authfile

### PR DESCRIPTION
**Pull request by AndresTanasijczuk**
This pull request is to allow LIGO DNs to read **all** LIGO private namespaces in the caches Authfile.
The caches Authfile is currently allowing LIGO DNs to read the `/user/ligo` private path, but not the other private paths recently added to the LIGO VO. See for example https://topology.opensciencegrid.org/cache/Authfile?fqdn=osg-new-york-stashcache.nrp.internet2.edu

PULL REQUEST NOT TESTED

### CODE REVIEW

File: src/stashcache.py
Comment (matyasselmeci) [inline src/stashcache.py:323]: Why are you removing this?

Comment (AndresTanasijczuk) [reply src/stashcache.py:323]: `generate_cache_grid_mapfile` is not using it, or am I wrong?

**Comment by josh-willis** [comment]
For now we (IGWN) would like to hold off on this, as we discuss internally what our short and medium term plans for X.509 proxies are.

**Comment by josh-willis** [comment]
@matyasselmeci @AndresTanasijczuk I'm happy for code development, but just to reiterate my point above that we don't want to actually merge this, pending an internal discussion among the LIGO and Virgo computing leadership. But we do appreciate the work on the pull request!

File: src/stashcache.py
Comment (matyasselmeci) [reply src/stashcache.py:323]: Ah, you're right. That's fine, then.

**Comment by bbockelm** [comment]
@josh-willis / @brianhlin -- should we revisit this one now?

**Comment by josh-willis** [comment]
> @josh-willis / @brianhlin -- should we revisit this one now?

@bbockelm Yes, please.

**Comment by brianhlin** [comment]
> Yes, please.

@josh-willis does this mean that IGWN approves of this change in principle?

**Comment by josh-willis** [comment]
Yes, it does

On Sep 9, 2023 06:46, Brian Lin ***@***.***> wrote:

Yes, please.

@josh-willis<https://github.com/josh-willis> does this mean that IGWN approves of this change in principle?

**Review by matyasselmeci — COMMENTED** [review]
This looks good to me; I will test it before approving and merging

**Review by matyasselmeci — APPROVED** [review]
My tests passed. This looks good to me.

File: src/tests/test_stashcache.py
Comment (matyasselmeci) [inline src/tests/test_stashcache.py:53]: Increase call count: a new LIGO namespace got added
```suggestion
        assert spy.call_count == 5
```

File: src/tests/test_stashcache.py
Comment (matyasselmeci) [inline src/tests/test_stashcache.py:60]: Increase call count: a new LIGO namespace got added
```suggestion
        assert spy.call_count == 5
```

**Comment by matyasselmeci** [comment]
Looks like a new IGWN namespace got added which changed the number of times get_ligo_dn_list() got called in the tests.  I'll merge anyway; we can improve the tests in a separate PR.

**Pull request merged**
